packages/notebook-extension/schema/tracker.json

```diff
@@ -699,6 +699,12 @@
       "type": "boolean",
       "default": false
     },
+    "showInputPlaceholder": {
+      "title": "Show input placeholder",
+      "description": "Show placeholder text for standard input fields. (Needs reload)",
```

Comment:
@JasonWeill what do you think would be the best wording here?

```suggestion
      "description": "Show placeholder text for standard input fields (requires reloading JupyterLab to take effect)",
```

Reply:
I think that (needs reload) or (requires reload) is both informative and concise. The longer phrase would be better for screen reader users, though.

Reply:
@krassowski Should I change the description?